### DISCUSSION TITLE
开始搞反动了狗汉奸？让网信办名正言顺封了github?gitee的倒钩狼？

### DISCUSSION THREAD

**Discussion by lovedeer4ever** [The-Run-Philosophy-Organization/run#168] [General]
真他妈晦气

**Comment by ghost** [comment]
Gitee：此文件因为违法相关法律法规而被屏蔽

**Comment by bulai0408** [comment]
就这格局属实让人有点绷不住

**Comment by ggymm** [comment]
确实，真他妈的晦气，污染Github环境

**Comment by Keith994** [comment]
都是神经病，要彻底封了github才行，狗汉奸真的狗

**Comment by ghost** [comment]
> 都是神经病，要彻底封了github才行，狗汉奸真的狗

这怎么就汉奸了，这怎么就危害国家安全了？
Github上比这个反动的项目多了去了，这个已经相当友好了吧
Github去年2月份就遭到SNI干扰了，这个项目的创立可是这个月的事情吧（

**Comment by godjoem** [comment]
> > 都是神经病，要彻底封了github才行，狗汉奸真的狗
> 
> 这怎么就汉奸了，这怎么就危害国家安全了？ Github上比这个反动的项目多了去了，这个已经相当友好了吧 Github去年2月份就遭到SNI干扰了，这个项目的创立可是这个月的事情吧（

Contributor哥，能不能专心搞润学方法论啊，搞这建政的东西别的地方很多啊，让润学纯粹一点呗

**Comment by ghost** [comment]
> > > 都是神经病，要彻底封了github才行，狗汉奸真的狗
> > 
> > 
> > 这怎么就汉奸了，这怎么就危害国家安全了？ Github上比这个反动的项目多了去了，这个已经相当友好了吧 Github去年2月份就遭到SNI干扰了，这个项目的创立可是这个月的事情吧（
> 
> Contributor哥，能不能专心搞润学方法论啊，搞这建政的东西别的地方很多啊，让润学纯粹一点呗

emm，我估计还会添加关于言论审查的部分（这下更不纯粹了
关于怎么润的部分，我不太清楚，需要其他人来更新（

**Comment by ghost** [comment]
不过我可以写一些关于计算机部分的内容（

**Comment by raceychan** [comment]
就算网信办因为这个repo把github封了，你不去骂网信办，骂这个repo？果然是五千年深入骨髓的奴性。

**Comment by cychan2047** [comment]
笑死。绑匪把人质杀了你在这里怪警察？

**Comment by bulai0408** [comment]
就算网信办因为这个repo把github封了，你不去骂网信办，骂这个repo？果然是五千年深入骨髓的奴性。

**Reply by leon-fudan** [reply]
连自己五千年祖宗都骂，你感觉学习好run掉吧。

**Reply by Ruth-Seven** [reply]
中国人怎么被人奴役了2000年都不敢承认？

**Comment by godjoem** [comment]
> 就i算网信办因为这个repo把github封了，你不去骂网信办，骂这个repo？果然是五千年深入骨髓的奴性。

虽然但是。github一个开源代码社区是不是还是少点带主观政治色彩的repo呢？😂

**Reply by ED-Y-S** [reply]
一句话：关你屁事。

**Reply by leon-fudan** [reply]
我看到了文明和粗鲁

**Comment by ghost** [comment]
> > 就i算网信办因为这个repo把github封了，你不去骂网信办，骂这个repo？果然是五千年深入骨髓的奴性。
> 
> 虽然但是。github一个开源代码社区是不是还是少点带主观政治色彩的repo呢？😂

但什么算涉及主观的政治内容呢？

[内宣这个文件夹是我创建的，但这算带主观政治色彩吗？](https://github.com/The-Run-Philosophy-Organization/run/tree/main/%E5%86%85%E5%AE%A3)

[外交部发言精选这个文件夹也是同样的问题，这个算带主观政治色彩吗？](https://github.com/The-Run-Philosophy-Organization/run/tree/main/%E5%A4%96%E4%BA%A4%E9%83%A8%E5%8F%91%E8%A8%80%E7%B2%BE%E9%80%89)

最后，送给在做的各位：朋友，这不是Github里搞政治，这是政治搞你，政治无处不在。

**Comment by godjoem** [comment]
> > > 就i算网信办因为这个repo把github封了，你不去骂网信办，骂这个repo？果然是五千年深入骨髓的奴性。
> > 
> > 
> > 虽然但是。github一个开源代码社区是不是还是少点带主观政治色彩的repo呢？😂
> 
> 但什么算涉及主观的政治内容呢？
> 
> [内宣这个文件夹是我创建的，但这算带主观政治色彩吗？](https://github.com/The-Run-Philosophy-Organization/run/tree/main/%E5%86%85%E5%AE%A3)
> 
> [外交部发言精选这个文件夹也是同样的问题，这个算带主观政治色彩吗？](https://github.com/The-Run-Philosophy-Organization/run/tree/main/%E5%A4%96%E4%BA%A4%E9%83%A8%E5%8F%91%E8%A8%80%E7%B2%BE%E9%80%89)
> 
> 最后，送给在做的各位：朋友，这不是Github里搞政治，这是政治搞你，政治无处不在。

其实我觉得这些可以单独建repo汇总，不过可能润学就是要伴随着建政内容吧。择需关注喽

**Reply by niuniuRun** [reply]
这个库如果只针对方法论来讲存在是什么任何问题的，但是其中如果掺杂了run的定义与社会现象就会带有个人的主观感情色彩，就会引起争端。

**Comment by SunsetMkt** [comment]
讨论 #153 也谈论了这个问题，这里已经出现人身攻击了，请注意GitHub社区守则

**Comment by williamtuw** [comment]
抓紧最后的时间clone 代码吧，我看这样下去不用几个星期 就要被墙了，以后没得用了，该死的反贼竟然这么多，快到7k的star了

**Comment by ghost** [comment]
> 抓紧最后的时间clone 代码吧，我看这样下去不用几个星期 就要被墙了，以后没得用了，该死的反贼竟然这么多，快到7k的star了

你想多了，Github上涉及政治的 用户/项目 多了去了，GFW也只是干扰了Github的连接

**Comment by smog150** [comment]
不要怪5000年的奴性，旧民主主义的梁、黄、孙，新民主主义的毛、周、邓这些先贤早就带领过人民破奴性；甚至是四人帮、十年前的胡温也不养这样的奴性；
也就13年开始吃包子饭的人会这样。

**Reply by liubaicai** [reply]
啥吃包子饭开始养奴性，根本就是从那时候开始，国家不在仅限于韬光养晦，美帝坐不住了而已。
在美帝眼里，南联盟南海撞机这种欺负完你你还无可奈何的中国才是好中国。

**Reply by ghost** [reply]
> 啥吃包子饭开始养奴性，根本就是从那时候开始，国家不在仅限于韬光养晦，美帝坐不住了而已。 在美帝眼里，南联盟南海撞机这种欺负完你你还无可奈何的中国才是好中国。

这倒是事实，但问题是为啥有人认为如果网信办封了Github应该来骂这个项目......

**Reply by leon-fudan** [reply]
这和俄罗斯打乌克兰，乌克兰人民很惨，但是罪魁祸首是谁呢？
就如台湾直接宣布独立，中国大陆动武，人民很惨，但是罪魁祸首是谁呢？
多思考一层，你可以看到，动用手段的不一定是坏人，义正词严谴责的，更不一定是好人，而有可能是罪魁祸首。

**Reply by ghost** [reply]
> 这和俄罗斯打乌克兰，乌克兰人民很惨，但是罪魁祸首是谁呢？ 就如台湾直接宣布独立，中国大陆动武，人民很惨，但是罪魁祸首是谁呢？ 多思考一层，你可以看到，动用手段的不一定是坏人，义正词严谴责的，更不一定是好人，而有可能是罪魁祸首。

不过为什么我国的态度是站Russia
不过为啥马英九都说一国两制已死？

所以判断罪魁祸首是不是应该知道事情的完整而不是片面呢？

**Reply by leon-fudan** [reply]
我和大多数国内程序员一样，有完整的全网access。如果你认为我的认知片面，可以把我没看过的内容发给我吗？谢谢。
不过我要说的，我国站Russia是基于目前的世界形势，站在自己国家发展的立场上，这并没有错啊。而马英九说什么，我可能确实不太关心。

**Reply by ghost** [reply]
> 我和大多数国内程序员一样，有完整的全网access。如果你认为我的认知片面，可以把我没看过的内容发给我吗？谢谢。 不过我要说的，我国站Russia是基于目前的世界形势，站在自己国家发展的立场上，这并没有错啊。而马英九说什么，我可能确实不太关心。

都是哪些人说的Russia倒下了下一个就到中国了........
如果下一个是中国，那战狼式外交有相当大的责任
